drivers/input/misc/fpc1145_platform.c

```diff
@@ -598,13 +598,15 @@ static int fpc1145_probe(struct platform_device *pdev)
 		goto exit;
 
 	hw_type = cei_fp_module_detect();
-	dev_info(dev, "Detected hw type %d\n", hw_type);
 
 	(void)vreg_setup(fpc1145, VDD_ANA, false);
 
 	if (hw_type != FP_HW_TYPE_FPC) {
+		dev_info(dev, "FPC sensor not found, bailing out\n");
 		rc = -ENODEV;
 		goto exit;
+	} else {
+		dev_info(dev, "Detected FPC sensor\n");
```

Comment:
nitpick, else is not needed because of that `goto exit;`

Reply:
Thanks, being a vast proponent of removing unnecessary cases to improve readability, I don't know how that slipped in :sweat_smile: